wp-content/themes/haruoshi_origin/src/scss/archive-news.scss

```
@@ -21,4 +21,24 @@
       margin-left: 0;
     }
   }
+  &__tag-list {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    margin-bottom: 30px;
+  }
+  &__tag {
+    border: solid 2px $accent-color;
+    background-color: $base-color;
+    border-radius: 100vh;
```

```suggestion
    border-radius: 10px;
```
ここの指定が若干気持ち悪けど見た目に変わりはないから許容

---

一応直してからマージ
https://github.com/oooshima/origin_wordpress/pull/34/commits/4ec76ae7854c8defe5742c6f2961ba67f6ae5f1f